docs/utils/index.rst

```diff
@@ -26,6 +26,7 @@ The exceptions are below:
 
    iers
    data
+   masked/index
```

Comment:
Should this be a separate folder, or just a file? Looking at the discussion, there are many followup PRs, e.g. for things like masked SkyCoord in Tables, but currently...

Reply:
I was indeed assuming it would be useful to make this a separate folder. For instance, on how to build your own container class that uses `Masked` etc.